docker-compose.yml

```diff
@@ -74,6 +74,10 @@ services:
       VALIDATOR_BASE_PATH: /validator
     mem_limit: 1000m
     restart: unless-stopped
+  inferno_bulk_data_server:
+    image: infernocommunity/bulk-data-server
+    mem_limit: 1500m
+    restart: unless-stopped
     volumes:
```

Comment:
There was definitely a merge mistake here, the bulk data server section seems to have stolen the 'volumes' section from the app above it.